app/api/resources/mentorship_relation.py

```diff
@@ -179,7 +179,7 @@ def get(cls):
         return response
 
 
-@mentorship_relation_ns.route("mentorship_relation/<int:request_id>/accept")
+@mentorship_relation_ns.route("mentorship_relation/<int:relation_id>/accept")
```

Comment:
No need to do any of the changes(i.e from request_id to relation_id) here. Since, we still haven't accepted the relation request.